axelrod/tests/unit/test_version.py

```diff
@@ -0,0 +1,11 @@
+"""
+Tests the version number
+"""
+from axelrod import __version__
+import unittest
+
+expected_version = "1.11.0"
+
+class TestVersion(unittest.TestCase):
+    def test_version(self):
+        self.assertEqual(__version__, expected_version)
```

Comment:
Yeah, I'm not terribly sure this was needed, mainly put it in as I was just checking everything was being imported:

1. I replace the test with a check that the attribute is there (so that `axelrod.__version__` exists basically)?
2. Remove it?